SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Lunar/RelicOfEchelon.cs

```diff
@@ -219,25 +189,28 @@ public void ModifyStatArguments(RecalculateStatsAPI.StatHookEventArgs args)
 
         public class EchelonToken : MonoBehaviour
         {
-            //helps keep track of the target and player responsible
-            //public CharacterBody PlayerOwner;
             public IDictionary<EquipmentIndex, int> usedEquipment;
             public CharacterBody body;
             public bool hadBuff;
 
-            //public void FixedUpdate()
-            //{
-            //    if()
-            //}
             public IEnumerator callRecalcOnBuffEnd()
             {
-                //SS2Log.Info("i have been awoken");
                 yield return new WaitForSeconds(buffDuration + .01f);
                 body.RecalculateStats();
-                //SS2Log.Info("zzzzzzzzzzzzzzzzzz....");
+            }
+        }
+
+        public sealed class EchelonBuffBehavior : BaseBuffBehaviour, IBodyStatArgModifier
```

Comment:
This buff behaviour can be replaced by a single call to REcalculateStatsAPI.

Reply:
Alright I just removed the BuffBehavior stat modifier because there seems to be a different stat modifier in the item itself. This code is really bad